src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/get-started.mdx

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
I don't see a purpose in this doc. I'd just cut it out entirely and make this the intro to the other doc